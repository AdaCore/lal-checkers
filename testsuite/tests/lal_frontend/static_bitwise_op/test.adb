procedure Test is
   A : constant := 2;
   B : constant := 4;
   X : Integer := A or B;
   Y : Integer := 7 and A;
begin
   null;
end Test;
