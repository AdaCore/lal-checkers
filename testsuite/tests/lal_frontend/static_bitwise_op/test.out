Program:
  X#1 = 6
  Y#2 = 2
  end: